cmd/dep/main.go

```diff
@@ -104,6 +126,12 @@ func (c *Config) Run() int {
 				fmt.Fprintf(tw, "\t%s\t%s\n", cmd.Name(), cmd.ShortHelp())
 			}
 		}
+		fmt.Fprintf(
+			tw,
+			"\t%s\t%s\n",
+			"constraints",
+			`Prints version constraints syntax`,
+		)
```

Comment:
Sorry for the confusion. Let's not do this since it's not like the other commands.